.travis.yml

```diff
@@ -0,0 +1,7 @@
+language: go
+go:
+- 1.9
+
+go_import_path: github.com/dachad/check-max-tcp-connections
```

Comment:
is this actually needed? Reading this (https://docs.travis-ci.com/user/languages/go/#Go-Import-Path) is not clear for me

Reply:
Because "vanity imports are necessary (especially for internal package imports), go_import_path: may be specified at the top level of the config", I needed to specify this here